README.md

```diff
@@ -4,7 +4,7 @@
 
 # Baseline Java code quality plugins
 [![CircleCI Build Status](https://circleci.com/gh/palantir/gradle-baseline/tree/develop.svg?style=shield)](https://circleci.com/gh/palantir/gradle-baseline)
-[![Bintray Release](https://api.bintray.com/packages/palantir/releases/gradle-baseline/images/download.svg) ](https://bintray.com/palantir/releases/gradle-baseline/_latestVersion)
+[![Gradle Plugin Portal](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/com/palantir/baseline-idea/com.palantir.baseline-idea.gradle.plugin/maven-metadata.xml.svg?label=plugin&logo=gradle)](https://plugins.gradle.org/plugin/com.palantir.baseline-idea)
```

Comment:
Thanks for fixing these links. Could we use https://plugins.gradle.org/plugin/com.palantir.baseline as the link here as it is the parent plugin?

Reply:
Fix in #1802